### PR TITLE
Feature/slim 2675 device authorization add not null constraints

### PR DESCRIPTION
**Pull request by kroesctrl**

### CODE REVIEW

File: osgp/platform/osgp-core/src/main/resources/db/migration/V20201020120000000__Add_not_null_constraints_to_device_authorization.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+DELETE FROM device_authorization WHERE device IS NULL OR organisation IS NULL;
```
Comment (harrymiddelburgetall) [inline osgp/platform/osgp-core/src/main/resources/db/migration/V20201020120000000__Add_not_null_constraints_to_device_authorization.sql:1]: Okay, als dit met Gerbert gecheckt is

**Review by harrymiddelburgetall — APPROVED** [review]

File: osgp/platform/osgp-core/src/main/resources/db/migration/V20201020120000000__Add_not_null_constraints_to_device_authorization.sql (hunk 1 above)
Comment (kroesctrl) [reply osgp/platform/osgp-core/src/main/resources/db/migration/V20201020120000000__Add_not_null_constraints_to_device_authorization.sql:1]: Yes, dit is gechecked met Gerbert en Maastricht

**Review by bvdzwet — COMMENTED** [review]
The changes that have been done look OK to me. I wonder whether it would be better to extend the scope of the story to also make function_group not null, by the same argumentation that looks to have been followed for making device and organisation not null.

**Comment by kroesctrl** [comment]
@bvdzwet Th function_group has been added as a NOT NULL column, could you please review the changes?

**Review by bvdzwet — APPROVED** [review]

**Pull request merged**